### PR TITLE
[Timezone] Support /etc/localtime as link

### LINKED ISSUE #59: [Timezone] Handle /etc/localtime
Should we copy from "/usr/share/zoneinfo/*/*" or create a symbolic link ???

### PR DESCRIPTION
**Pull request by nervo**
Fix #59

### CODE REVIEW

File: manala.timezone/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Support debian /etc/localtime as link (as a consequence, "manala_timezone_file" and "manala_timezone_template" variables becomes irrelevants)
```
Comment (chalasr) [inline manala.timezone/CHANGELOG.md]: `become` (plural) and `irrelevant`

File: manala.timezone/tasks/default.yml [hunk 2]
```diff
@@ -1,12 +1,34 @@
...
-  when: (manala_timezone_template is not none) or (manala_timezone_default is not none)
+- name: default > Localtime stat
+  stat:
+    path: /etc/localtime
+  when:
+    - manala_timezone_default is not none
+  register: __manala_timezone_default_localtime
+
+# If "/etc/localtime" is a regular file, the good old strategy consists in
+# editing "/etc/timezone" and reconfigure tzdata package...
```
Comment (chalasr) [inline manala.timezone/tasks/default.yml]: reconfiguring

File: manala.timezone/tasks/default.yml [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
+    dest: /etc/timezone
+    content: >
+      {{ manala_timezone_default|ternary(manala_timezone_default, 'Etc/UTC') }}
+  when:
+    - not __manala_timezone_default_localtime|skipped
+    - __manala_timezone_default_localtime.stat.isreg
+  notify:
+    - dpkg-reconfigure tzdata
+
+# ...on the contrary, if it's a link, we need to update it before reconfigure tzdata
```
Comment (chalasr) [inline manala.timezone/tasks/default.yml]: reconfiguring

**Comment by nervo** [comment]
@chalasr , fixed, thx !

**Pull request merged**